### ISSUE TITLE
Inference on custom dataset

### ISSUE THREAD

**Issue by gokcergun** [qklee-lz/ACMMM2024-MAC#1]
Hi, 

Can we use your model to perform inference for micro action recognition on a custom dataset?

Thank you.

**Comment by qklee-lz** [comment]
Of course! If you find it useful, feel free to cite our paper. Thank you!

**Comment by tatban** [comment]
Hi @qklee-lz, I am also interested to use your model on custom data. Is there anywhere pre-trained weights available for your model?  The checkpoint links in the different readme files don't seem to work. It would be really nice if you can point me to some of the working pre-trained weight checkpoints. Thanks

**Comment by qklee-lz** [comment]
@tatban Hi ~ I checked the link and made sure it doesn't work now... The pre-trained weights we used are listed in our paper, I think you can according to the paper and then search on hugging face.
